www/docs/style/sass/pages/_home.scss

```diff
@@ -327,10 +327,10 @@ input.homepage-search__button {
 
 .excerpt__title {
     margin-bottom: 0.1em;
-    max-width: 21em;
     font-size: 1.125em;
     @media (min-width: $medium-screen) {
         font-size: 1em;
+        padding-right: 10em;
```

Comment:
Not sure this was enough:
![screen shot 2018-12-20 at 09 49 39](https://user-images.githubusercontent.com/154364/50277553-9e2fea80-043c-11e9-86ba-31616351ab87.png)

Reply:
Ah poop, didn’t spot "Written ministerial statements"!